docs/resources/cortex_search_service.md

```diff
@@ -77,6 +75,16 @@ resource "snowflake_cortex_search_service" "test" {
 - `created_on` (String) Creation date for the given Cortex search service.
 - `id` (String) The ID of this resource.
 
+<a id="nestedblock--timeouts"></a>
+
+### Nested Schema for `timeouts`
+
+Optional:
+
+- `create` (String)
```

Comment:
The changes in this file seem to have been added manually. Please revert them and run `make pre-push` following the contribution guidelines: https://github.com/Snowflake-Labs/terraform-provider-snowflake/blob/main/CONTRIBUTING.md#before-submitting-the-pr (because this file is generated).

Reply:
Done - ran `make pre-push` which updated these docs.